lib/flash.js

```diff
@@ -0,0 +1,184 @@
+var async = require('async');
+var PAGE = 256;
+var logs = require('./logs');
+var CHIP_ID = new Buffer([0x01, 0x02, 0x19]);
```

Comment:
Take a look at the deps section in other files—they are split into categories and separate from other var decls. I'm not going to block this patch on that, as I can just do a fix up after it lands, but good to know for future reference :)